uv.lock

```diff
@@ -0,0 +1,2094 @@
+version = "14.0.1.1.1.dev2"
+source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "odoo" },
+    { name = "odoo14-addon-sql-request-abstract" },
+]
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/59/50/f000ba229579551433381148c64644e5cd5fa590be3c95b5c3fcf5810b46/odoo14_addon_bi_sql_editor-14.0.1.1.1.dev2-py3-none-any.whl", hash = "sha256:c96be6ffb09a4ea4de396bf52eaa85d646b917addd15831a2fa2e6947d1c56ea", size = 619782 },
+]
+
+[[package]]
+name = "odoo14-addon-currency-rate-update"
+version = "14.0.1.3.3.dev7"
+source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "odoo" },
+]
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/01/c4/9c5f1619203862f2faa6be68fe2325523952d6274bdd7b594913899ff4ec/odoo14_addon_currency_rate_update-14.0.1.3.3.dev7-py3-none-any.whl", hash = "sha256:87644c0c52884e6bb507d1d9a7d37dbfe718f4c9b9d499f28caad7e2fde63593", size = 248976 },
+]
+
+[[package]]
+name = "odoo14-addon-date-range"
+version = "14.0.2.2.0"
+source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "odoo" },
+]
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/c2/8f/e1d198f8c2f0439bc0370702342e4fe1360fb9db09e0d7b4e013b0189d7a/odoo14_addon_date_range-14.0.2.2.0-py3-none-any.whl", hash = "sha256:c2309176c5fb349e593cd0cc9433fb34b37c4d82a0562057e3360e45e2c5e6a4", size = 1147819 },
+]
+
+[[package]]
+name = "odoo14-addon-disable-odoo-online"
+version = "14.0.1.0.1.dev6"
+source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "odoo" },
+]
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/f9/44/a9a103db372bc0e42d20e1f6e12a54ffbddd8ee0330abb87771d26deb72e/odoo14_addon_disable_odoo_online-14.0.1.0.1.dev6-py3-none-any.whl", hash = "sha256:14a478150aa07c50a9fded1259639d8e80b39929c6141c3e03b38a3c0a3e68d6", size = 25020 },
+]
+
+[[package]]
+name = "odoo14-addon-document-page"
+version = "14.0.1.3.2.dev2"
+source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "odoo" },
+    { name = "odoo14-addon-knowledge" },
+]
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/d4/57/b7252a92cb7760835c7cf41bbb662b2121f49f209efb12aadbc26fbab859/odoo14_addon_document_page-14.0.1.3.2.dev2-py3-none-any.whl", hash = "sha256:c3703d415dcf552ce5d7867e8ed3cb63968c53954aeb39099257c57dca33380b", size = 933268 },
+]
+
+[[package]]
+name = "odoo14-addon-event-badge-design"
+version = "14.0.1.0.2.dev5"
+source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "odoo" },
+]
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/5c/67/ed36ceff8cf235700e796936bb5347ec6497dbd11f8dcef0d1f27ad9d40b/odoo14_addon_event_badge_design-14.0.1.0.2.dev5-py3-none-any.whl", hash = "sha256:708845cd8ef5cd50aa8337f25b63e680b1092cc64f77a68d3a6ab61cb103be65", size = 22033 },
+]
+
+[[package]]
+name = "odoo14-addon-github-connector"
+version = "14.0.2.3.1.dev1"
```

Comment:
@sbidoul For all `oca_all` module dependencies, I initially used the latest versions available on PyPI when running `uv sync` for the first time. Would you like me to gently freeze them (only in `uv.lock`) to match the versions used in production, so that a plain `uv sync` ensures exact same dependencies, while `uv sync -U` allows updates?

Reply:
It's probably not worth the effort. I suspect they have not changed that much since they were last updated in prod. So let's try with the latest versions.